TestLeaderStopAndReElectWithPriority unittest run failure

It happens frequently, maybe we have to investigate it.

```
testLeaderStopAndReElectWithPriority(com.alipay.sofa.jraft.core.NodeTest)  Time elapsed: 7.238 sec  <<< FAILURE!
java.lang.AssertionError: expected:<60> but was:<10>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.alipay.sofa.jraft.core.NodeTest.testLeaderStopAndReElectWithPriority(NodeTest.java:1051)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```

--- a/jraft-core/src/test/java/com/alipay/sofa/jraft/core/NodeTest.java
+++ b/jraft-core/src/test/java/com/alipay/sofa/jraft/core/NodeTest.java
@@ -1060,7 +1060,7 @@ public class NodeTest {
         if (leaderPriority == 10) {
             // we just compare the two peers' log size value;
             assertTrue(peer2LogSize > peer1LogSize);
-        }else {
+        } else {
             assertEquals(60, leader.getNodeId().getPeerId().getPriority());
             assertEquals(100, leader.getNodeTargetPriority());
         }

--- a/jraft-core/src/test/java/com/alipay/sofa/jraft/core/NodeTest.java
+++ b/jraft-core/src/test/java/com/alipay/sofa/jraft/core/NodeTest.java
@@ -1048,8 +1048,22 @@ public class NodeTest {
         leader = cluster.getLeader();
 
         assertNotNull(leader);
-        assertEquals(60, leader.getNodeId().getPeerId().getPriority());
-        assertEquals(100, leader.getNodeTargetPriority());
+
+        // get current leader priority value
+        int leaderPriority = leader.getNodeId().getPeerId().getPriority();
+
+        // get current leader log size
+        int peer1LogSize = cluster.getFsmByPeer(peers.get(1)).getLogs().size();
+        int peer2LogSize = cluster.getFsmByPeer(peers.get(2)).getLogs().size();
+
+        // if the leader is lower priority value
+        if (leaderPriority == 10) {
+            // we just compare the two peers' log size value;
+            assertTrue(peer2LogSize > peer1LogSize);
+        }else {
+            assertEquals(60, leader.getNodeId().getPeerId().getPriority());
+            assertEquals(100, leader.getNodeTargetPriority());
+        }
 
         cluster.stopAll();
     }

